internal/oscap/main.go

```diff
@@ -0,0 +1,45 @@
+package oscap
+
+import "strings"
+
+type Profile string
+
+func (p Profile) toString() string {
```

Comment:
If you rename this method to `String()`, it gets called automatically when you format it with `fmt.Print()` and its variants (it implements the [Stringer](https://pkg.go.dev/golang.org/x/tools/cmd/stringer) interface), which could be useful.